project/Dependencies.scala

```diff
@@ -40,7 +40,7 @@ object Dependencies {
   private val ficusV = "1.5.1"
   // The "com.vladsch.flexmark" % "flexmark-profile-pegdown" % flexmarkV dependency is an implicit, version-specific
   // runtime dependency of ScalaTest. At the time of this writing this is the newest version known to work.
-  private val flexmarkV = "0.36.8" // scala-steward:off
+  private val flexmarkV = "0.62.2" // scala-steward:off
```

Comment:
can the `scala-steward:off` go away now?

Reply:
No, I don't think that's a good idea. If we try to upgrade flexmark to a version beyond what Scalatest supports, we could see errors. Scalatest and flexmark should be upgraded together, which I tried to explain with my inline comment.